plugins/RedditPlugin/.env.dist

```diff
@@ -0,0 +1,4 @@
+REDDIT_CLIENT_ID=
```

Comment:
Add this to the main .env, rather than having it's own seperate .env

Reply:
Resolved at https://github.com/lavenduale/GarfieldBot/commit/b961e2211fe22551c4c58f58fb6844c1b2f2a6a0